### PR TITLE
Add post to releases, go to 0.2

### PR DESCRIPTION
**Pull request by ripleyaffect**
- Add a post to `CONTENT_BUNDLES_URL/{curriculum.id}/releases` to notify of new releases, if set
- Upload to a new path, with just the curriculum id, instead of the code and version

The curriculum id here is the combined code and version

### CODE REVIEW

File: src/lib/getCurriculumFromModule.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 const processMarkdown = require('./processMarkdown');
 
+const getId = modFile =>
+  `${modFile.code}-v${modFile.version || 1}`.toLowerCase();
```
Comment (learykara) [inline src/lib/getCurriculumFromModule.js]: why are all of the version fallbacks necessary everywhere? why not just use what's there?

File: src/lib/postCurriculumRelease.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    };
+
+    log(`Posting content bundle "${curriculum.id}" to ${url}`);
+
+    response = await axios.post(url, data, {
+      params: { token: CONTENT_BUNDLE_RELEASES_AUTH_TOKEN },
+    });
+
+    log(
+      c.green(`Successfully posted content bundle "${curriculum.id}" to ${url}`)
```
Comment (learykara) [inline src/lib/postCurriculumRelease.js]: is this guaranteed successful bc of await & try/catch?

Comment (ripleyaffect) [reply src/lib/postCurriculumRelease.js]: yeah, an error response will throw

File: src/lib/getCurriculumFromModule.js (hunk 1 above)
Comment (ripleyaffect) [reply src/lib/getCurriculumFromModule.js]: Just a precaution against bad data that gets past

**Pull request merged**